specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/WebApps.json

```diff
@@ -21567,6 +21567,412 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/deployWorkflowArtifacts": {
+      "post": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Creates the artifacts for web site, or a deployment slot.",
+        "description": "Description for Creates the artifacts for web site, or a deployment slot.",
+        "operationId": "WebApps_DeployWorkflowArtifacts",
+        "parameters": [
+          {
```

Comment:
I notice you're describing this action as "Create". Do you also support "Update", or "Delete"? Why aren't you modeling it as a PUT request?

Reply:
Good question! Due to how logic apps works, each workflow is initialized with an empty definition, and there is only one deploy command to update that definition with a whole new file. This is the same way our other deployment methods work across logic apps, functions, app service, etc. 

I added another example to this API, since it's also used to delete workflows with a slightly different payload.